filter_plugins/dashboards.py

```diff
@@ -17,20 +17,12 @@ def extract_new_dashboards(dashboard_meta, existing_dashboard_meta):
     """
     existing_dashboard_titles = {d['title'] for d in json.loads(
                                           existing_dashboard_meta['content'])}
-    available_dashboards = _load_dashboards(dashboard_meta)
+    available_dashboards = [json.loads(
+                       base64.b64decode(db['content'])) for db in dashboards]
```

Comment:
Does this support updating existing dashboards?

Reply:
I've made it so if a dashboard exists with a given title, it's not updated if we try and load one with the same title. I thought about updating regardless, but I was worried about the risk of peoples tweaks getting overwritten before they'd had a chance to update the dashboard repo. So in summary, if you want to update your dashboard, delete it first from the Grafana GUI and then this will upload the latest version.